paasta_tools/cli/cmds/validate.py

```diff
@@ -203,6 +204,11 @@ def validate_chronos(service_path):
     chronos_spacer = paasta_tools.chronos_tools.INTERNAL_SPACER
 
     returncode = True
+
+    if service == TMP_JOB_IDENTIFIER:
```

Comment:
I was expecting us to not allow `.startswith(TMP_JOB_IDENTIFIER)` (as opposed to the `==` you've got here)?